tools/python/checkupstreamversion.py

```diff
@@ -0,0 +1,37 @@
+import json
```

Comment:
```suggestion
#!/usr/bin/env python

import json
```

Reply:
Sure I've also pruned the un-necessary `import json`